module/x/gravity/types/ethereum_event.go

```diff
@@ -131,7 +118,7 @@ func (stce *SendToCosmosEvent) Validate() error {
 	if !common.IsHexAddress(stce.TokenContract) {
 		return sdkerrors.Wrap(ErrInvalid, "ethereum contract address")
 	}
-	if !stce.Amount.IsPositive() {
+	if !stce.Amount.IsNegative() {
```

Comment:
this is a bug being introduced. Please revert

Reply:
Well what we want is for all non negative numbers to be OK and only negative numbers to error. This code should do that right?

Reply:
The old code was also a bug which would let any Ethereum user halt the bridge by sending one transaction on Ethereum. 

But we accidently left on the !, which would be a bug which would prevent the bridge from working as well. Good catch.